### PR TITLE
Yaml serialized questions

### LINKED ISSUE #174: Add yaml input for questions
This will make it easy to add new questions.  It would be nice if the questions could also be saved into a yaml file.

### PR DESCRIPTION
**Pull request by maheshgudi**
- Yaml serialisation instead of Json for dumping and uploading questions.
- Can download a yaml template for creating questions within the file and uploading it on the interface.
- UI Changes on the question section.
- closes #174

### CODE REVIEW

File: yaksh/live_server_tests/selenium_test.py [hunk 1]
```diff
@@ -115,6 +116,17 @@ def open_quiz(self):
         self.test_python_question(question_label=5)
         self.test_bash_question(question_label=4)
 
+    def quit_quiz(self):
+        quit_link_elem = WebDriverWait(self.driver, 5).until(
```
Comment (prabhuramachandran) [inline yaksh/live_server_tests/selenium_test.py]: This wait business can be encapsulated into a function so you don't need to keep repeating the same `WebDriverWait(self.driver, 5).until(...)`

File: yaksh/demo_templates/yaml_question_template [hunk 2]
```diff
@@ -0,0 +1,462 @@
+---
```
Comment (prabhuramachandran) [inline yaksh/demo_templates/yaml_question_template]: This template is a bit backwards. I'd like the description to be first and the testcase last, perhaps we should dump the dict in sorted order, maybe use an ordered dict?

Comment (maheshgudi) [inline yaksh/demo_templates/yaml_question_template]: Resolved. The dict is dumped into yaml in alphabetical order.

File: yaksh/live_server_tests/selenium_test.py (hunk 1 above)
Comment (maheshgudi) [inline yaksh/live_server_tests/selenium_test.py]: Duly noted. Will do in an upcoming PR. Resolving for now.

File: yaksh/demo_templates/yaml_question_template (hunk 2 above)
Comment (prabhuramachandran) [inline yaksh/demo_templates/yaml_question_template]: Can you please update this template as well to be alphabetically sorted?

**Comment by prabhuramachandran** [comment]
@maheshgudi -- please update the template checked in to be alphabetically sorted.

File: yaksh/demo_templates/yaml_question_template (hunk 2 above)
Comment (maheshgudi) [inline yaksh/demo_templates/yaml_question_template]: Resolved

**Pull request merged**